### PR TITLE
Shapes SVG include PHP Parse Error #9693 #9694 : check file exist

### PR DESCRIPTION
**Pull request by MarieComet**
Original PR : https://github.com/elementor/elementor/pull/9694
Requested change by @bainternet

### CODE REVIEW

**Review by bainternet — DISMISSED** [review]

File: includes/elements/section.php [hunk 1]
```diff
@@ -1528,9 +1528,13 @@ private function print_shape_divider( $side ) {
 		$settings = $this->get_active_settings();
 		$base_setting_key = "shape_divider_$side";
 		$negative = ! empty( $settings[ $base_setting_key . '_negative' ] );
+		$shape_path = Shapes::get_shape_path( $settings[ $base_setting_key ], $negative );
+		if ( ! file_exists( $shape_path ) ) {
```
Comment (matipojo) [inline includes/elements/section.php]: Check also it's not a URL.

```suggestion
		if ( ! is_file(  $shape_path ) ||  ! file_exists( $shape_path ) ) {
```

**Review by matipojo — APPROVED** [review]

**Review by matipojo — APPROVED** [review]

**Comment by KingYes** [comment]
Thanks

**Comment by MarieComet** [comment]
What do you think about `is_readable` instead of `file_exists` ?
`is_readable` : "Tells whether a file exists **and is readable**"
`file_exists` : "Checks whether a file or directory exists"

**Comment by matipojo** [comment]
@MarieComet You are right `is_readable` is better.

**Pull request merged**